cl/lib/search_utils.py

```diff
@@ -432,27 +433,23 @@ def add_highlighting(main_params, cd, highlight):
         fl = ['absolute_url', 'citeCount', 'court_id', 'dateFiled',
               'download_url', 'id',  'local_path', 'sibling_ids', 'source',
               'status']
-        hlfl = ['caseName', 'citation', 'court_citation_string', 'docketNumber',
-                'judge', 'lexisCite', 'neutralCite', 'suitNature', 'text']
+        hlfl = settings.SOLR_OPINION_HL_FIELDS
```

Comment:
These should either follow the approach of the items above or the items above should be moved to settings, but splitting like this makes me sad.